style/buttons.css

```
@@ -0,0 +1,114 @@
+.button{
+    font-family: 'Georgia', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 22.1px;
+    letter-spacing: 0.06em;
+    border-radius: 9px;
+    cursor: pointer;
+    background: var(--main-bg-color);
+    color: var(--text-color-black);
+    padding: 5px 10px 5px 10px;
```

Could be simplified a bit
```suggestion
    padding: 5px 10px;
```